src/Groups/gsets.jl

```diff
@@ -450,17 +450,28 @@ julia> map(length, orbs)
 """
     stabilizer(Omega::GSet{T,S})
     stabilizer(Omega::GSet{T,S}, omega::S = representative(Omega); check::Bool = true) where {T,S}
+    stabilizer(Omega::GSet{T,S}, omega::Set{S}; check::Bool = true) where {T,S}
+    stabilizer(Omega::GSet{T,S}, omega::Vector{S}; check::Bool = true) where {T,S}
+    stabilizer(Omega::GSet{T,S}, omega::Tuple{Vararg{S}}; check::Bool = true) where {T,S}
 
 Return the subgroup of `G = acting_group(Omega)` that fixes `omega`,
 together with the embedding of this subgroup into `G`.
+
+If `omega` is a `Set` or a `Vector` or a `Tuple` of points in `Omega`
+then `stabilizer` means the setwise or pointwise stabilizer, respectively,
+of the entries in `omega`.
```

Comment:
I had to read this sentence multiple times to understand it. Could you maybe split the setwise and the pointwise part into two different sentences?

Reply:
I actually feel like this is clear as written. Does it really need to be split up?

Reply:
the ambiguity I see is that some of set, vector and tuple have setwise and some have pointwise stabilizers. But this sentence does not tell me which has which.
A sentence like in the PR description avoids this:
> The stabilizer of a Set of points in the G-set is the setwise stabilizer. The stabilizer of a Vector or Tuple of points in the G-set is the pointwise stabilizer.